lib/shiftzilla/org_data.rb

```diff
@@ -40,7 +40,7 @@ def populate_releases
 
           # TODO: REMOVE BUSINESS LOGIC EMBEDDED IN CODE
           if comp == 'Security' and keyw.include?('Unconfirmed')
-            puts "NB: This report has a hardcoded exclusion of 'Security' component bugs with the 'Unconfirmed' keyword."
+            #puts "NB: This report has a hardcoded exclusion of 'Security' component bugs with the 'Unconfirmed' keyword."
```

Comment:
Even though it is commented out as a command, please remove the `puts` so this reads like a comment.

Reply:
Done.